### ISSUE TITLE
Not working on Anki 2.1.44 for Windows

### ISSUE THREAD

**Issue by Bortex19** [Arthur-Milchior/anki-resize-image#40]
It just does nothing

**Comment by Arthur-Milchior** [comment]
Can you please give me your configuration file - if you did not keep default.ç
I've tested with .43, I didn't see that 44 was out, i'll try.

I don't have a windows, but I doubt it change a lot of thing.

**Comment by Bortex19** [comment]
Hi, Arthur! 
Thank you for answer.

{
    "0": 0,
    "apply maximum height when not resizing": false,
    "apply maximum width when not resizing": false,
    "apply minimum height": true,
    "apply minimum width": true,
    "draggable border width": 15,
    "image-classes": [],
    "max-height": 200,
    "max-width": 200,
    "min-height": 10,
    "min-width": 10,
    "note-type style": ".mobile .card img {height:unset  !important; width:unset  !important;}",
    "preserve ratio while resizing": "current",
    "resizable-style": "border:1px dashed white;"
}

I have restored the defaults and have uninstalled and reinstalled the add-on. I disabled other similar add-ons but nothing worked. I can't figured out what is happening.

44 version was just to fix minor things.


    In case this was useful to you:
    Anki 2.1.44 (b2b3275f) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
    Platform: Windows 10
    Flags: frz=True ao=True sv=2
    Add-ons, last update check: 2021-05-23 21:41:00
    
    
    ===Add-ons (active)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    Add Table ['1237621971', 2020-07-25T14:19, 'None', mod]
    Advanced Browser ['874215009', 2021-02-21T15:38, 'None', '']
    Anki Simulator ['817108664', 2021-02-21T16:44, 'None', '']
    Batch Editing ['291119185', 2020-01-12T19:30, 'None', '']
    Cloze Overlapper for Anki 21 ['109531687', 2021-01-12T04:12, 'None', '']
    Fastbar- with nightmode support ['46611790', 2021-05-19T02:50, 'None', '']
    Hierarchical Tags 2 ['594329229', 2021-03-07T03:25, 'None', '']
    Hint Hotkeys ['1844908621', 2017-08-29T20:53, 'None', '']
    Image Occlusion Enhanced for Anki 21 alpha ['1374772155', 2020-04-28T04:21, 'None', '']
    More Overview Stats 21 ['738807903', 2021-02-05T18:35, 'None', mod]
    No Distractions Full Screen Clean review interface  TabletTouch support ['1049863218', 2020-08-14T00:59, 'v4.1.8', mod]
    Pop-up Dictionary beta ['153625306', 2019-06-20T20:15, 'None', '']
    Postpone cards review ['1152543397', 2021-02-13T17:47, 'None', '']
    Progress Bar ['2091361802', 2020-01-12T18:07, 'None', '']
    Progress Graphs and Stats for Learned and Matured Cards ['266436365', 2020-03-29T08:26, 'None', '']
    Resize images in editor ['1103084694', 2021-05-23T15:09, 'None', mod]
    Review Heatmap ['review_heatmap', 2018-10-28T23:06, 'None', '']
    Speed Focus Mode auto-alert auto-reveal auto-fail ['1046608507', 2019-12-10T16:38, 'None', '']
    Spelling Police ['1410276506', 2020-03-03T03:13, 'None', mod]
    Symbols As You Type ['2040501954', 2021-03-22T03:17, 'None', '']
    The KING of Button Add-ons ['374005964', 2020-04-18T00:33, 'None', mod]
    True Retention by Card Maturity ['923360400', 2019-11-27T20:31, 'None', '']
    Web Browser - Search terms Import texts and images automatically ['864545277', 2020-08-29T20:23, 'None', mod]
    Ze Frozen Fields ['94610912', 2021-02-12T01:06, 'None', '']
    extended editor for field for tables searchreplace  ['805891399', 2020-11-24T01:08, 'None', '']
    load balancer ['1417170896', 2020-08-06T01:58, 'None', '']
    
    ===IDs of active AnkiWeb add-ons===
    1046608507 1049863218 109531687 1103084694 1152543397 1237621971 1374772155 1410276506 1417170896 153625306 1844908621 2040501954 2091361802 266436365 291119185 374005964 46611790 594329229 738807903 805891399 817108664 864545277 874215009 923360400 94610912
    
    ===Add-ons (inactive)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    Color Confirmation ['1084228676', 2020-03-14T13:13, 'None', '']
    Enhance main window ['877182321', 2020-12-23T05:42, 'None', '']
    Frozen Fields ['516643804', 2020-03-01T19:28, 'None', '']
    Image Editor ['307397307', 2021-02-24T10:41, 'None', mod]
    Image Style Editor ['1593969147', 2020-12-17T14:06, 'None', mod]
    Large and Colorful Buttons ['1829090218', 2018-10-02T19:41, 'None', '']
    Leaderboard ['41708974', 2020-12-10T10:25, 'None', mod]
    Mini Format Pack ['295889520', 2018-07-24T19:45, 'None', '']

**Comment by HarryLang100** [comment]
Hmm, this is very odd. At first, it does not work. However, if I change the card type while there is an image, it does work just like it used to.

**Comment by Arthur-Milchior** [comment]
I put an update one or two hours ago, can you please try again?

Can you maybe take a video screenshot? or just tell me what you're doing to test the add-on?

**Comment by HarryLang100** [comment]
Hello. I have attached a video screenshot to this post. Does that explain what I'm on about?

https://user-images.githubusercontent.com/10622188/119402494-ebcb7400-bcd4-11eb-8d63-78b94286ddb6.mp4

**Comment by Bortex19** [comment]
> I put an update one or two hours ago, can you please try again?
> 
> Can you maybe take a video screenshot? or just tell me what you're doing to test the add-on?

I can use this workaround now, but it does not work as it should.

And thank you for your job, this add-on is really useful.

**Comment by Arthur-Milchior** [comment]
Yeah, I get it. I forgot to check what occur if you add a new image. I was only testing editing already existing image.
It will wait, I've no mental energy for going back to it right now

**Comment by Bortex19** [comment]
Take your time, it is usable by the moment 😀

**Comment by HarryLang100** [comment]
Thanks for working on this, I really appreciate it :)
